### ISSUE TITLE
rebase --onto

### ISSUE THREAD

**Issue by clounie** [ForkIssues/Tracker#126]
A relatively common pattern is to push a hotfix into `master` when something's broken - but only once we've confirmed that it works in `dev`. I tend to do it something like this:

```bash
# 1. Checkout the feature branch which you based on dev
git checkout hotfix-branch-on-dev
git pull

# 2. Make a pull request branch for master, based on the dev feature branch
git checkout -b hotfix-PR-for-master

# 3. Pull the commits from your dev feature branch, 
#      into the PR branch intended for master
git rebase --onto master dev hotfix-PR-for-master
```

I realize this might be getting into the weeds a little bit, but it would be nice to have this option in Fork.

**Comment by DanPristupov** [comment]
I didn't know about the `--onto` flag previously. It's kind of Cherry-Pick but for many commits, right?

Currently I don't see a way to fit this feature into Fork's UI. But it could be possible when I implement merging/rebasing by drag'n'drop. User can just select a sequence of commits and drag them on another branch, which will start `rebase` with the `--onto` flag.

**Comment by clounie** [comment]
Yep, basically; that'd be great.

**Comment by DanPristupov** [comment]
This can be done using commit targeted Custom Actions which were introduced in 1.0.67.

**Comment by DanPristupov** [comment]
@YesThatAllen sorry I missed your question. What you are looking for is a simple rebase:
<img width="376" alt="2018-07-16 at 22 33" src="https://user-images.githubusercontent.com/618115/42782110-40181a5e-8948-11e8-92bf-b202d660c746.png">
